src/artm/messages.proto

```diff
@@ -846,3 +846,8 @@ message TransformMasterModelArgs {
   optional ThetaMatrixType theta_matrix_type = 3 [default = Dense];
   optional string predict_class_id = 4;
 }
+
+message ConfigureLoggingArgs {
```

Comment:
> the level can be any int, not only 0-3
Ok, I agree -- then enum might not be good idea. Let's keep int. Maybe you may copy a line from glog help into messages.proto just before the `minloglevel` field.
```// The numbers of severity levels INFO, WARNING, ERROR, and FATAL are 0, 1, 2, and 3, respectively.```